docs/en/adfs.md

```diff
@@ -50,6 +51,24 @@ Click the "Details" tab and click "Copy to file".
 
 A wizard opens, click "Next" and then choose "Base-64 encoded X.509 (.CER)". Click "Next" and choose a place to export the certificate. Click "Next" to finish the process.
 
+## Create a new relying party trust
+
```

Comment:
Sorry for deleting :-| @halkyon would be neat if you could add here the "why", or what it actually accomplishes. Something like "here is where we authorise the SilverStripe site as a party that can request identity information from this IdP" or whatever.